R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1448,23 +1457,28 @@ test_that("write.df() on DataFrame and works with read.parquet", {
   expect_equal(count(df), count(parquetDF))
 })
 
-test_that("read.parquet()/parquetFile() works with multiple input paths", {
+test_that("read.parquet()/parquetFile() and write.parquet()/saveAsParquetFile()", {
```

Comment:
How about renaming this test case to "read/write Parquet files", where read.parquet(), parquetFile(), read.df(), write.parquet(), saveAsParquetFile(), write.df() are tested to read/write json files. This test case can be merged with the one "write.df() as parquet file"